src/Curl/Curl.php

```diff
@@ -75,7 +75,9 @@ class Curl
     private $responseCookies = array();
     private $headers = array();
     private $options = array();
-
+    
+    private $responseParsing = null;
```

Comment:
I would rather see this as `protected`, allowing an inheriting class to set it as part of it's definition.